fastlane/Allurefile

```diff
@@ -17,7 +17,7 @@ end
 
 desc 'Create launch on Allure TestOps'
 lane :allure_launch do |options|
-  next unless is_check_required(sources: sources_matrix[:e2e])
+  # next unless is_check_required(sources: sources_matrix[:e2e])
```

Comment:
Do we want to comment this?

Reply:
Oh, good catch 🙈. I was testing around. Will revert this shortly

Reply:
Done